src/exposurelog/shared_state.py

```diff
@@ -110,16 +109,21 @@ def __init__(self) -> None:
                 f"SITE_ID={site_id!r} too long; max length={SITE_ID_LEN}"
             )
 
-        self.registries: list[lsst.daf.butler.Registry] = []
```

Comment:
Please remove `registries` attribute from the `SharedState` class docstrings. Also `butler_factory` attribute should be added.